config.sh

```diff
@@ -26,15 +26,17 @@ LIBWEBP_VERSION=1.2.4
 BZIP2_VERSION=1.0.8
 LIBXCB_VERSION=1.14
 
-if [[ -n "$IS_MACOS" ]] && [[ "$PLAT" == "x86_64" ]]; then
-    function build_openjpeg {
-        local out_dir=$(fetch_unpack https://github.com/uclouvain/openjpeg/archive/v${OPENJPEG_VERSION}.tar.gz)
-        (cd $out_dir \
-            && cmake -DCMAKE_INSTALL_PREFIX=$BUILD_PREFIX -DCMAKE_INSTALL_NAME_DIR=$BUILD_PREFIX/lib . \
-            && make install)
-        touch openjpeg-stamp
-    }
-fi
+function build_brotli {
+    local cmake=$(get_modern_cmake)
+    local out_dir=$(fetch_unpack https://github.com/google/brotli/archive/v1.0.9.tar.gz)
```

Comment:
Should we add this version as a constant above?

Do we still need `OPENJPEG_VERSION` there?

Reply:
Ok, I've added a `BROTLI_VERSION` variable.

Yes, `OPENJPEG_VERSION` is still used by multibuild in https://github.com/python-pillow/pillow-wheels/blob/8c65e05daae37986c92d35242aa0a2f60971f0a1/config.sh#L88